js/utilities.js

```diff
@@ -731,6 +731,44 @@ var ERMrest = (function(module) {
 
             return module._markdownIt.render(value);
         },
+        
+        /**
+        * @function		    
+        * @param {Object} value A json value to transform
+        * @param {Object} [options] Configuration options.
+        * @return {string} A string representation of value
+        * @desc Formats a given json value into a string for display.
+        */
+        printJSON: function printJSON(value, options) {
+           options = (typeof options === 'undefined') ? {} : options;
+           if(options.context==="entry/edit"){
+               if (value === null) {
+                   return 'null';
+                }
+                else if (typeof value === 'object') {
+                       return JSON.stringify(value,undefined,2);
+                }
+                else if (typeof value === 'string') {
```

Comment:
You don't need any of these `if` statements. `JSON.stringify` returns the correct value in all of the cases. just `return JSON.stringify(value, undefined, 2);`.

Also change this to check for any entry contexts. Although we are never going to call this function in `entry/create`, but it doesn't make sense to add `<pre>` tags to any entry contexts, as these contexts are suppose to show the "raw" value.

```javascript
if (module._isEntry(options.context)) {
  return JSON.stringify(value, undefined, 2);
}
```